src/util/sienna/helper_functions.jl

```diff
@@ -187,3 +221,18 @@ function get_outage_time_series_data(
 
     return λ_gen, μ_gen
 end
+
+function make_area_interchange_dict(area_interchanges::Vector{PSY.AreaInterchange})
+    area_interchange_data = Dict{String, PSY.FromTo_ToFrom}()
+    for a_i in area_interchanges
+        f_l = PSY.get_flow_limits(a_i)
+        push!(
+            area_interchange_data,
+            PSY.get_name(PSY.get_from_area(a_i)) *
```

Comment:
I don't think you actually need to use the string "=>" since you just index the dictionary with a pair of strings.